UnityProject/Assets/Scripts/Health/Living/LivingHealthBehaviour.cs

```diff
@@ -1048,13 +1054,49 @@ public string GetExamineText()
 			if (this is PlayerHealth && GetComponent<PlayerScript>().mind.IsOnline() == false)
 			{
 				healthString += $"\n{theyPronoun} has a blank, absent-minded stare and appears completely unresponsive to anything. " +
-						$"{theyPronoun} may snap out of it soon.";
+				                $"{theyPronoun} may snap out of it soon.";
 			}
 		}
 
 		return healthString;
 	}
 
+	public string GetShortStatus()
+	{
+		if (IsDead)
+		{
+			return "DEAD";
```

Comment:
could add extra if for is SSD and dead?

Reply:
Yes I can. I didn't know if we wanted to add information that specific, is it in character information to know how someone is dead and has no soul?

Reply:
done